src/elements/content-uploader/ItemRemove.js.flow

```diff
@@ -34,25 +29,23 @@ const ItemRemove = ({ intl, onClick, status }: Props) => {
         resin['data-resin-target'] = target;
     }
 
+    const intl = useIntl();
     const isDisabled = status === STATUS_STAGED;
     const tooltipText = intl.formatMessage(messages.remove);
 
     return (
         <div className="bcu-item-action">
-            <Tooltip position="top-left" text={tooltipText}>
-                <PlainButton
+            <Tooltip content={tooltipText} variant="standard">
+                <IconButton
                     aria-label={tooltipText}
-                    isDisabled={isDisabled}
+                    disabled={isDisabled}
                     onClick={onClick}
-                    type="button"
+                    icon={() => <XMark color="black" height={Size5} width={Size5} />}
```

Comment:
is color here just the css color field? if so can we use the blueprint variable for black ?